API/Keybindings.cs

```diff
@@ -200,14 +199,14 @@ public Keybinding(KeybindingDescription description)
     private void ComputeInputFlag()
     {
         var idBytes = Encoding.UTF8.GetBytes(Description.Id);
-        var hash = (long)xxHash64.ComputeHash(idBytes, idBytes.Length);
+        var hash = xxHash64.ComputeHash(idBytes, idBytes.Length);
         var invalid = false;
         do
         {
             invalid = false;
-            foreach (var entry in Enum.GetValues(typeof(InputFlag)).Cast<long>())
+            foreach (var entry in Enum.GetValues(typeof(InputFlag)))
             {
-                if (hash == entry)
+                if (hash == (ulong)entry.GetHashCode())
```

Comment:
This doesn't have the same behavior, use the generic version of Enum.GetValues and cast to `ulong` for clarity.
```cs
foreach (var entry in Enum.GetValues<InputFlag>())
{
    if (hash == (ulong)entry)
    {
        invalid = true;
        hash -= 1;
    }
}
```